### PR TITLE
Fix quantized example

### PR DESCRIPTION
**Pull request by lucasavila00**
I found it while testing https://github.com/EricLBuehler/mistral.rs/pull/236

### CODE REVIEW

File: mistralrs-core/src/sampler.rs [hunk 1]
```diff
@@ -35,6 +35,23 @@ pub struct SamplingParams {
...
+            temperature: None,
+            top_k: None,
+            top_p: None,
+            top_n_logprobs: 0,
+            frequency_penalty: None,
+            presence_penalty: None,
+            stop_toks: None,
+            max_len: None,
+            logits_bias: None,
+            n_choices: 1,
```
Comment (lucasavila00) [inline mistralrs-core/src/sampler.rs:50]: it was defaulting to 0

Comment (lucasavila00) [reply mistralrs-core/src/sampler.rs:50]: I'm not sure if this should use the Completion Request `best_of` instead. These 2 sources of truth are confusing

Comment (EricLBuehler) [reply mistralrs-core/src/sampler.rs:50]: @lucasavila00 the `best_of` parameter sets the number of seqs to consider when they are done by cumulative logprob. It should be <= `n_choices`.

**Review by EricLBuehler — APPROVED** [review]

**Comment by EricLBuehler** [comment]
Thank you

**Pull request merged**